k8s/us-central1/gen/tf-nightly-xlnet-squad-conv-v3-8.yaml

```diff
@@ -53,7 +53,6 @@
             - "--test_feature_path=$(XLNET_SQUAD_DIR)/spiece.model.slen-512.qlen-64.eval.features.pkl"
             - "--predict_file=$(XLNET_SQUAD_DIR)/dev-v2.0.json"
             - "--predict_dir=$(MODEL_DIR)"
-            - "--train_batch_size=48"
```

Comment:
Is this change intentional?

Reply:
Yes, thanks for checking! I had --train_batch_size set in the model command but I prefer it with the accelerator mixin.